src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -12,4 +12,5 @@ public Truck() {
     public void doWork() {
         System.out.println("Truck started to work");
     }
+
```

Comment:
The task description requires adding custom fields that are specific to a truck. Consider adding fields such as `cargoCapacity`, `numberOfAxles`, or any other attributes that are unique to a truck.